src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -1724,8 +1724,11 @@ private void DoRunspaceInitialization(bool importSystemModules, bool skipProfile
                 if (_configuration != null)
                     shellId = _configuration.ShellId;
                 else
+#if CORECLR                
+                    shellId = Platform.IsInbox ?  "Microsoft.PowerShell" : "PowerShell";
+#else
```

Comment:
I'm not sure about the necessity to make this change. I just put a comment at #1190 for @mirichmo to review and response.